docs: fleet setup scripts (#716)
Azure/fleet: hack/Azure/setup/README.md

# Setting up a Fleet

> This how-to guide discusses how to create a fleet, specifically:

Reviewer: ```suggestion
This how-to guide describes how to create a fleet using Azure Kubernetes Service, specifically:

```

> * how to create a hub cluster; and

Reviewer: ```suggestion
* how to create an AKS cluster as the  hub cluster; and

```

> * how to join clusters to hub cluster; and

Reviewer: ```suggestion
* how to join any k8s clusters to the AKS hub cluster; and

```

> * how to add labels to a member cluster

Reviewer: ```suggestion
* how to add labels to a member cluster representation on the hub cluster

```
